torch/testing/_internal/distributed/rpc/dist_autograd_test.py

```diff
@@ -78,14 +78,20 @@ def create_tensor():
     return torch.ones((3, 3), requires_grad=True)
 
 
-def build_sparse_tensor(coalesce=False, requires_grad=True, dtype=torch.float32):
+def build_sparse_tensor(coalesce=False, requires_grad=True, dtype=torch.float32, device=None):
     i = [[0, 1, 1], [2, 0, 2]]
     v = [3.2, 4.1, 5.3]
-    tensor = torch.sparse_coo_tensor(i, v, (3, 3), requires_grad=requires_grad, dtype=dtype)
+    tensor = torch.sparse_coo_tensor(i, v, (3, 3), requires_grad=requires_grad, dtype=dtype, device=device)
     if coalesce:
         tensor = tensor.coalesce()
     return tensor
 
+def build_sparse_one_gradient(dtype=torch.float32):
```

Comment:
looks like this is only used once, maybe does not worth to make it a function?

Reply:
It is used two times. I would have to add the def to the test, so I am going to keep it.